modules/calib3d/include/opencv2/calib3d.hpp

```diff
@@ -747,7 +771,8 @@ the board to make the detection more robust in various environments.
  */
 CV_EXPORTS_W bool findCirclesGrid( InputArray image, Size patternSize,
                                    OutputArray centers, int flags = CALIB_CB_SYMMETRIC_GRID,
-                                   const Ptr<FeatureDetector> &blobDetector = SimpleBlobDetector::create());
+                                   const Ptr<FeatureDetector> &blobDetector = SimpleBlobDetector::create(),
+                                   CirclesGridFinderParameters parameters = CirclesGridFinderParameters());
```

Comment:
Hey @StevenPuttemans , thank you for the review. I have adjusted the PR accordingly. Could you verify this is as you intended?